packages/vaadin-combo-box/src/vaadin-combo-box-dropdown.js

```diff
@@ -20,6 +20,8 @@ registerStyles(
   { moduleId: 'vaadin-combo-box-overlay-styles' }
 );
 
+const ONE_THIRD = 0.3;
```

Comment:
**nit:** Would `1 / 3` work here?

Reply:
I am not sure if it is Ok to change this value as it's also used in `_shouldAlignAbove` 🤔 
It won't probably be noticeable, but it's still changing the default.

Reply:
Oh, got it.